_includes/header.html

```diff
@@ -9,56 +9,56 @@
       <ul class="nav navbar-nav mr-auto">
         <li class="nav-item dropdown">
             <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">About us<span class="caret"></span></a>
-            <ul class="dropdown-menu" style="background-color: #000e33;">
-                <li style="background-color: #000e33;" class="dropdown-item{% if page.url == "/about/" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/index.html">The FAMNM</a></li>
-                <li style="background-color: #000e33;" class="dropdown-item{% if page.url == "/about/core.html" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/leadership.html">Core Team</a></li>
-                <li style="background-color: #000e33;" class="dropdown-item{% if page.url == "/about/org-structure.html" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/org-structure.html">Organizational Structure</a></li>
+            <ul class="dropdown-menu" style="background-color: #003368;">
+                <li style="background-color: #003368;" class="dropdown-item{% if page.url == "/about/" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/index.html">The FAMNM</a></li>
+                <li style="background-color: #003368;" class="dropdown-item{% if page.url == "/about/core.html" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/leadership.html">Core Team</a></li>
+                <li style="background-color: #003368;" class="dropdown-item{% if page.url == "/about/org-structure.html" %} active{% endif %}"><a class="nav-link" href="{{site.url}}/about/org-structure.html">Organizational Structure</a></li>
```

Comment:
{nit} Have you considered using Custom CSS vars so that we can make this change once instead? Example here https://www.w3.org/TR/css-variables/#defining-variables

Reply:
Note that this could be used to simplify any where we want our custom palette used. 
Define things like famnm-blue, famnm-white, etc.
Maybe more a follow up cleaning of this.

Reply:
i'm in favor of this; however, i believe that refactoring the CSS will be a separate tech debt PR (we have a honkin 400+ line `main.css` and we need to consolidate and re-evaluate it) outside of this.